test/sparse/cholmod.jl

```diff
@@ -436,7 +436,7 @@ for elty in (Float64, Complex{Float64})
 
     ## Low level interface
     @test CHOLMOD.nnz(A1Sparse) == nnz(A1)
-    @test CHOLMOD.speye(5, 5, elty) == eye(elty, 5, 5)
+    @test CHOLMOD.speye(5, 5, elty) == Matrix(I, 5)
```

Comment:
Probably does not matter, but perhaps `Matrix{elty}(I, 5)`

Reply:
In `==`, the `Matrix(I, ...)` on the right-hand-side lifts to the eltype of the left-hand-side, making the eltype specification on the right unnecessary :).